jobs/build/ocp/Jenkinsfile

```diff
@@ -378,14 +373,29 @@ node(TARGET_NODE) {
             }
 
             stage( "puddle: ose 'building'" ) {
-                buildlib.build_puddle(
-                        PUDDLE_CONF,    // The puddle configuration file to use
-                        "-b",   // do not fail if we are missing dependencies
-                        "-d",   // print debug information
-                        "-n",   // do not send an email for this puddle
-                        "-s",   // do not create a "latest" link since this puddle is for building images
-                        "--label=building"   // create a symlink named "building" for the puddle
-                )
+                if ( SIGN_RPMS ) {
+                  buildlib.build_puddle(
+                          "--conf"
```

Comment:
Does not align with the parameters of buildlib.build_puddle. Happens to work, but we should not need to specify --conf, or --keys for each invocation. Keep things DRY as well.

```
keys = SIGN_RPMS?"b906ba72":null
buildlib.build_puddle( PUDDLE_CONF, keys, -b, .....)
```

This will require a signature change to build_puddle method. build_puddle( conf, keys, Object... args). Method implementation could include "--conf" and "--keys" if parameter is not null.